eq3bt/eq3cli.py

```diff
@@ -21,16 +21,17 @@ def validate_mac(ctx, param, mac):
 
 @click.group(invoke_without_command=True)
 @click.option('--mac', envvar="EQ3_MAC", required=True, callback=validate_mac)
+@click.option('--interface', default=False)
```

Comment:
Without digging into the source of the bluepy, should the default rather be None? The PR looks otherwise straightforward to me.

Reply:
Yes, you are absolutely right! Sorry about that.